README.md

```diff
@@ -17,33 +17,46 @@ That's why we've created smaller projects, almost independent mini-indexers:
 - Separate projects are easier to maintain;
 - The main difference is in the future: we are thinking where to go next if we decide to get rid of Postgres.
 
-### Why do the projects _almost_ independent?
-
-We still hope to leave foreign keys in the tables.
-The data provided by all the indexers depend on Indexer Base.  
-All the indexers may have the dependency to Indexer Accounts, but it will give us circular dependency, that's why we don't use these constraints.
-
 ### Can I create my own indexer?
 
 Sure!
 Feel free to use this project as the example.
 
 ## Linux installation guide
 
+Your `.env` file should be in the project root (`near-microindexers` folder) with the contents:
+```
+AWS_SECRET_ACCESS_KEY=...
+AWS_ACCESS_KEY_ID=...
+DATABASE_URL=...
+RPC_URL=...
+CHAIN_ID=mainnet
+INDEXER_ID=indexer-events-tip
+INDEXER_TYPE=indexer-events
+START_BLOCK_HEIGHT=30181671
+```
+
+- You need to have your own [AWS credentials](https://docs.near.org/tutorials/indexer/credentials) and pay for the S3 requests [around $20 per month](https://github.com/near/near-lake-framework-rs#cost-estimates);
+- You need to create the DB where the data will be stored; you also need to apply the migrations manually.
+- RPC URL could be found [here](https://docs.near.org/api/rpc/providers)
+- CHAIN_ID could be `mainnet` or `testnet`
+- INDEXER_ID could be anything; when you restart your app, it will find the last tracked `block_height` by this id;
+- INDEXER_TYPE could be also anything, but I suggest you to choose between the folders listed in this repo;
+- START_BLOCK_HEIGHT could be any non-negative integer; if the process is restarted, we ignore this field;
+
 ```bash
 sudo apt install git build-essential pkg-config libssl-dev tmux postgresql-client libpq-dev -y
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 source $HOME/.cargo/env
 cargo install --version=0.5.13 sqlx-cli --features postgres
 ulimit -n 30000
 cargo build --release
-#!!! here you need to create .env in the root of the project, and .aws in ~
-cargo run --release -- --s3-bucket-name near-lake-data-mainnet --s3-region-name eu-central-1 --start-block-height 9820210
+cargo run --release
 ```
 
 ## Migrations
```

Comment:
Doesn't have to be in this ticket, but would be good to have a "genesis" DB (In this case maybe a PostgreSQL dump) with at least 2022 data to avoid reprocessing on each installation